modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/components/fragments/Forms/Select.js

```diff
@@ -21,6 +21,7 @@ export const Select = React.forwardRef(
           className="input"
           required={required}
         >
+          <option hidden>Select</option>
```

Comment:
I think you should not add this to the `Select` atom. 

The `Select` fragment is an atomic component that represents a pure select input reusable by any component. Logic belongs outside the `fragment` as a `connector` or `container` component. 

I would move the `<option hidden>Select</option>` were it is needed.

Reply:
Ok!

Reply:
Done!